client/my-sites/marketplace/marketplace-domain-upsell/index.tsx

```diff
@@ -30,15 +30,15 @@ import Item from 'calypso/layout/masterbar/item';
 import getPreviousPath from 'calypso/state/selectors/get-previous-path';
 import { HorizontalRule } from '@wordpress/components';
 import ExternalLink from 'calypso/components/external-link';
-import MarketplaceShoppingCart from 'calypso/my-sites/plugins/marketplace/components/marketplace-shopping-cart';
-import theme from 'calypso/my-sites/plugins/marketplace/theme';
-import { MarketplaceHeaderTitle } from 'calypso/my-sites/plugins/marketplace/components';
-import { setPrimaryDomainCandidate } from 'calypso/state/plugins/marketplace/actions';
+import MarketplaceShoppingCart from 'calypso/my-sites/marketplace/components/marketplace-shopping-cart';
+import theme from 'calypso/my-sites/marketplace/theme';
+import { MarketplaceHeaderTitle } from 'calypso/my-sites/marketplace/components';
+import { setPrimaryDomainCandidate } from 'calypso/state/marketplace/purchase-flow/actions';
 
 /**
  * Style dependencies
  */
-import 'calypso/my-sites/plugins/marketplace/marketplace-domain-upsell/style.scss';
+import 'calypso/my-sites/marketplace/marketplace-domain-upsell/style.scss';
```

Comment:
Nit: style imports can be relative, and they usually are because the style file is in the same directory as the component. Totally optional and a matter of personal taste.

```suggestion
import './style.scss';
```